### DISCUSSION TITLE
"Pure" HTML/CSS still on the roadmap?

### DISCUSSION THREAD

**Discussion by chadgates** [vmware/clarity#6354] [Q&A] [answered]
Am unclear if the roadmap for V5 still holds pure HTML/CSS as was available in CLR/V4. Was planning to use Clarity a while ago for a project based on pure HTML (minimal JS with HTMX.org) instead of using a large JS Framework. 
If so, any guidance on how to get there?

**Comment by hippee-lee** [comment, marked as answer]
We do have a roadmap for the future of Clarity on the [website](https://clarity.design/get-started/roadmap/).

## RE: @clr/angular@v5
The @clr/angular (and dependency's `@clr/ui & @clr/icons` will be updated for the foreseeable future. In the last major version bump, we updated the `@clr/*` packages to v12 because it mirrors the compatible version of Angular and makes it easier to know which version of Clarity works with which version of Angular. 

We will continue to fix bugs per the [support policies ](https://clarity.design/get-started/support/)as outlinedon the website. 

## HTMX.org
As long as that package supports web-components the best thing to build with from Clarity will be the `@cds/core` web components that we offer. Most of the roadmap is around that and the future of Clarity.

**Reply by chadgates** [reply]
Thanks for your reply which helped me investigate in the right direction which led me to find this with no build step: 
 https://github.com/vmware/clarity/blob/next/apps/core-cdn-demo/index.html

**Reply by hippee-lee** [reply]
Sure thing. If you are looking for links to a running stackblitz for that app and more check out https://cds.wtf/ that @coryrylan built.

**Reply by coryrylan** [reply]
The https://cds.wtf/ site itself an example of using plain CSS/JS to build out a site. No build tools were used, just CDN links.
